MANIFEST.in

```
@@ -0,0 +1,29 @@
+include locations/searchable_points/au_centroids_120km_radius.csv
```

i am wondering if we should just include all of `locations/searchable_points` instead of hardcoding each manually, lest we forget to add this and wonder why it fails to find it when we add additional reference here.

---

We 100% should glob, I didn't know that was a thing, I've never written a `MANIFEST`

---

i think it's `graft locations/searchable_points`

---

though glob should also work!